test/cypress/integration/request-to-go-live/organisation_name_spec.js

```diff
@@ -80,11 +49,33 @@ describe('Request to go live: organisation name page', () => {
     })
   })
 
-  describe('Service has correct go live stage and organisation name is not pre-filled', () => {
+  describe('Service has invalid go live stage', () => {
+    const serviceRole = utils.buildServiceRoleForGoLiveStage('INVALID_GO_LIVE_STAGE')
+    beforeEach(() => {
+      utils.setupStubs(serviceRole)
+    })
+    it('should redirect to "Request to go live: index" page when in wrong stage', () => {
+      const requestToGoLivePageOrganisationNameUrl = `/service/${serviceExternalId}/request-to-go-live/organisation-name`
+      cy.visit(requestToGoLivePageOrganisationNameUrl)
+
+      cy.get('h1').should('contain', 'Request to go live')
+
+      cy.location().should((location) => {
+        expect(location.pathname).to.eq(`/service/${serviceExternalId}/request-to-go-live`)
+      })
+    })
+  })
+
+  describe('Service has NOT_STARTED go live stage and organisation name is not pre-filled', () => {
+    const serviceRolesBefore = utils.buildServiceRoleForGoLiveStage('NOT_STARTED')
+    const serviceRolesAfter = lodash.concat(
+      utils.buildServiceRoleForGoLiveStage('ENTERED_ORGANISATION_NAME'),
+      utils.buildServiceRoleForMerchantDetailsField({ name: undefined }))
```

Comment:
I'm not sure this works, as the method only expects a single service role.

Reply:
This was a slightly hacky attempt to make it clear that merchant details are not pre-filled. Also, thanks for clarifying the `validServiceResponse` default values in `service_fixtures` earlier,  i didn't notice it at all. When I see a code-block end `}` I still assume the method is done.